extensions/2.0/Vendor/GRIFFEL_bim_data/README.md

```diff
@@ -0,0 +1,222 @@
+# GRIFFEL\_bim\_data
+
+## Contributors
+
+* Kiryl Ambrazheichyk, Griffel Studio LLC, [kiryl@griffelstudio.com](mailto:kiryl@griffelstudio.com)
+* Alexey Knyazev, The Khronos Group, [@lexaknyazev](https://github.com/lexaknyazev)
```

Comment:
The "Contributors" section is supposed to represent ownership of the spec and/or provide point of contact for potential implementors, so please do not add me here.

Acknowledging the Khronos Group review feedback in GitHub comments is more than enough.

Reply:
That's right, ok. Removed.